Sources/SmokeDynamoDB/AWSDynamoDBCompositePrimaryKeyTable+updateItems.swift

```diff
@@ -203,6 +203,44 @@ public extension AWSDynamoDBCompositePrimaryKeyTable {
         return EventLoopFuture.andAllSucceed(futures, on: self.eventLoop)
     }
     
+    func monomorphicBulkWriteWithFallback<AttributesType, ItemType>(_ entries: [WriteEntry<AttributesType, ItemType>]) -> EventLoopFuture<Void> {
+        // fall back to singel operation if the write entry exceed the statement length limitation
+        var nonBulkWriteEntries: [WriteEntry<AttributesType, ItemType>] = []
+        var bulkWriteEntries: [WriteEntry<AttributesType, ItemType>] = []
+        do {
+            bulkWriteEntries = try entries.compactMap { entry in
+                do {
+                    try self.validateEntry(entry: entry)
```

Comment:
Is our problem with the size of a single entry or adding multiple entries together?

Reply:
Our problem is a single statement could exceed the limitation which is 8192